### PR TITLE
release v1.1.0-rc.2

### PR DESCRIPTION
**Pull request by cofyc**
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-1.1.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+## Other Notable Changes
+
+- Add `status` field for `TidbAutoScaler` CR ([#2182](https://github.com/pingcap/tidb-operator/pull/2182), [@Yisaer](https://github.com/Yisaer))
+- Add `spec.pd.maxFailoverCount` field to limit max failover replicas for PD ([#2184](https://github.com/pingcap/tidb-operator/pull/2184), [@cofyc](https://github.com/cofyc))
+- Emit more events for `TidbCluster` and `TidbClusterAutoScaler` to help users know TiDB running status ([#2150](https://github.com/pingcap/tidb-operator/pull/2150), [@Yisaer](https://github.com/Yisaer))
+- Add the `AGE` column to show creation timestamp for all CRDs. ([#2168](https://github.com/pingcap/tidb-operator/pull/2168), [@cofyc](https://github.com/cofyc))
+- Add a switch to skip PD Dashboard TLS configuration ([#2143](https://github.com/pingcap/tidb-operator/pull/2143), [@weekface](https://github.com/weekface))
+- Change TiDB Pod Readiness Probe from `HTTPGet` to `TCPSocket` 4000 port ([#2139](https://github.com/pingcap/tidb-operator/pull/2139), [@weekface](https://github.com/weekface))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:12]: This PR title uses "readness". Which one is correct?
```suggestion
- Change TiDB Pod Readness Probe from `HTTPGet` to `TCPSocket` 4000 port ([#2139](https://github.com/pingcap/tidb-operator/pull/2139), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+- Deploy TiFlash with TidbCluster CR ([#2157](https://github.com/pingcap/tidb-operator/pull/2157), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- TLS support for TiKV metrics API ([#2137](https://github.com/pingcap/tidb-operator/pull/2137), [@weekface](https://github.com/weekface))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:14]:
```suggestion
- Add TLS support for TiKV metrics API ([#2137](https://github.com/pingcap/tidb-operator/pull/2137), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+
+- Add `status` field for `TidbAutoScaler` CR ([#2182](https://github.com/pingcap/tidb-operator/pull/2182), [@Yisaer](https://github.com/Yisaer))
+- Add `spec.pd.maxFailoverCount` field to limit max failover replicas for PD ([#2184](https://github.com/pingcap/tidb-operator/pull/2184), [@cofyc](https://github.com/cofyc))
+- Emit more events for `TidbCluster` and `TidbClusterAutoScaler` to help users know TiDB running status ([#2150](https://github.com/pingcap/tidb-operator/pull/2150), [@Yisaer](https://github.com/Yisaer))
+- Add the `AGE` column to show creation timestamp for all CRDs. ([#2168](https://github.com/pingcap/tidb-operator/pull/2168), [@cofyc](https://github.com/cofyc))
+- Add a switch to skip PD Dashboard TLS configuration ([#2143](https://github.com/pingcap/tidb-operator/pull/2143), [@weekface](https://github.com/weekface))
+- Change TiDB Pod Readiness Probe from `HTTPGet` to `TCPSocket` 4000 port ([#2139](https://github.com/pingcap/tidb-operator/pull/2139), [@weekface](https://github.com/weekface))
+- Deploy TiFlash with TidbCluster CR ([#2157](https://github.com/pingcap/tidb-operator/pull/2157), [@DanielZhangQD](https://github.com/DanielZhangQD))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:13]:
```suggestion
- Support deploying TiFlash with TidbCluster CR ([#2157](https://github.com/pingcap/tidb-operator/pull/2157), [@DanielZhangQD](https://github.com/DanielZhangQD))
```

File: CHANGELOG-1.1.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+- TLS support for TiKV metrics API ([#2137](https://github.com/pingcap/tidb-operator/pull/2137), [@weekface](https://github.com/weekface))
+- Set PD Dashboard Config when TLS Client enabled ([#2085](https://github.com/pingcap/tidb-operator/pull/2085), [@weekface](https://github.com/weekface))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:15]:
```suggestion
- Set PD DashboardConfig when TLS between the MySQL client and TiDB server is enabled ([#2085](https://github.com/pingcap/tidb-operator/pull/2085), [@weekface](https://github.com/weekface))
```

File: CHANGELOG-1.1.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+- Remove unnecessary informer caches to reduce the memory footprint of tidb-controller-manager ([#1504](https://github.com/pingcap/tidb-operator/pull/1504), [@aylei](https://github.com/aylei))
+- Fix the failure that Helm cannot load the kubeconfig file when deleting the tidb-operator release during `terraform destroy` ([#2148](https://github.com/pingcap/tidb-operator/pull/2148), [@DanielZhangQD](https://github.com/DanielZhangQD))
+- Make Webhook tls configuration setting by loading secret. ([#2135](https://github.com/pingcap/tidb-operator/pull/2135), [@Yisaer](https://github.com/Yisaer))
```
Comment (ran-huang) [inline CHANGELOG-1.1.md:18]:
```suggestion
- Support configuring the Webhook TLS setting by loading a secret ([#2135](https://github.com/pingcap/tidb-operator/pull/2135), [@Yisaer](https://github.com/Yisaer))
```

File: CHANGELOG-1.1.md (hunk 1 above)
Comment (cofyc) [reply CHANGELOG-1.1.md]: "readiness" is correct, see https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

PR title fixed

File: CHANGELOG-1.1.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
+# TiDB Operator v1.1.0-rc.2 Release Notes
+
+This is the second release candidate of `v1.1.0`, which focuses on the usability, extensibility and security of TiDB Operator. While we encourage usage in non-critical environments, it is **NOT** recommended to use this version in critical environments.
+
+## Other Notable Changes
```
[the remark this answers is not in the document]
Comment (cofyc) [reply CHANGELOG-1.1.md]: updated

**Pull request merged**